### PR TITLE
Info card styling alignment

### PR DESCRIPTION
**Pull request by seanforyou23**
## Description
During work on [issue-111](https://github.com/patternfly/patternfly-ng/issues/111) in the patternfly-ng repo we noticed there was a styling discrepancy between the info status card examples. After speaking with a member of the design team it was decided to add an additional 15px of bottom margin to this component.

The SVG width/height was added because there was a bug where SVG's with no width/height defined within the xml didn't display at all. This gives them bounds of their own even when not specified.
screen shot 2017-10-31 at 10 40 33 am
screen shot 2017-10-31 at 10 40 43 am

## PR Checklist

- [ ] Unit tests are included
- [x] Screenshots are attached (if there are visual changes in the UI)
- [x] A Designer (@beanh66) is assigned as a reviewer (if there are visual changes in the UI)
- [x] A CSS rep (@cshinn) is assigned as a reviewer (if there are visual changes in the UI)

### CODE REVIEW

File: src/table/tableview/examples/table-view-basic.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
   * <ul style='list-style-type: none'>
   *   <li>.header     - (string) Text label for a column header
   *   <li>.itemField    - (string) Item field to associate with a particular column.
-  *   <li>.templateFn - (function) (optional) Template function used to render each cell of the column. Pro: more performant than `htmlTemplate`. Con: doesn't support AngularJS directives in the template, therefore it doesn't support things like ng-click. Example: <pre>templateFn: value => `<span class="text-danger">${value}</span>`</pre>
+  *   <li>.templateFn - (function(value, item)) (optional) Template function used to render each cell of the column. Pro: more performant than `htmlTemplate`. Con: doesn't support AngularJS directives in the template, therefore it doesn't support things like ng-click. Example: <pre>templateFn: (value, item) => `<a href="${item.id}">${value}</a>`</pre>
```
Comment (dtaylor113) [inline src/table/tableview/examples/table-view-basic.js]: Think you need to rebase.  These tableView changes should not be part of this PR.

**Comment by jeff-phillips-18** [comment]
The checkboxes for designer and css reviewers are checked but they were not added as reviewers.

**Comment by cshinn** [comment]
From a visual standpoint, I agree this is better. There are some issues with the status card itself that should be addressed either here or in a separate PR. The class names don't follow our pf-postfix standards for instance

**Comment by beanh66** [comment]
@seanforyou23 these styling changes look good to me, but as discussed I think we decided to bubble this up to an issue in PF core.

File: src/table/tableview/examples/table-view-basic.js (hunk 1 above)
Comment (seanforyou23) [reply src/table/tableview/examples/table-view-basic.js]: Understood, thanks for pointing this out. After speaking with some designers it looks like this issue is best addressed in PF core, so I believe this PR will be closed anyway. Let me know if there is anything else needed from me.
